lib/sparkperf/testsuites.py

```diff
@@ -251,8 +251,8 @@ class MLlibTests(JVMPerfTestSuite, MLlibTestHelper):
     test_jar_path = "%s/mllib-tests/target/mllib-perf-tests-assembly.jar" % PROJ_DIR
 
     @classmethod
-    def build(cls):
-        run_cmd("cd %s/mllib-tests; %s clean assembly" % (PROJ_DIR, SBT_CMD))
+    def build(cls, spark_version):
+        run_cmd("cd %s/mllib-tests; %s -Dspark.version=%s.0 clean assembly" % (PROJ_DIR, SBT_CMD, spark_version))
```

Comment:
Well, we kind of hack our way through by having `MLLIB_SPARK_VERSION` be a float, so the change wouldn't be trivial...

I think this is fine for now, given that we have yet to add features which need perf testing in patch versions. If the need arises, adding a `MLLIB_SPARK_PATCH_VERSION` to `config.py` will not be too hard.